src/controllers/carpool.ctrl.ts

```diff
@@ -96,12 +96,16 @@ module CarpoolController {
   export function getCarpools(
     req:restify.Request, res:restify.Response, next:restify.Next) {
     function getCarpoolList() {
-      var radius:number = parseFloat(req.query.radius);
-      var long:number = parseFloat(req.query.long);
-      var lat:number = parseFloat(req.query.lat);
       var campusName:string = req.query.campusName;
 
-      return carpoolService.getCarpools(10, radius, long, lat, campusName);
+      if(req.query.radius && req.query.long && req.query.lat) {
```

Comment:
Since we are dealing with numbers this should probably check if they are  not undefined instead of true.  In the somewhat rare event that someone is on the equator or the prime meridian this test will not pass because 0 is falsy.  Sorry if I'm being pedantic.